designer/server/src/main/scala/pl/touk/nussknacker/ui/server/NuAkkaHttpServerInterpreterForTapirPurposes.scala

```diff
@@ -0,0 +1,40 @@
+package pl.touk.nussknacker.ui.server
+
+import com.typesafe.scalalogging.LazyLogging
+import sttp.tapir.server.akkahttp.{AkkaHttpServerInterpreter, AkkaHttpServerOptions}
+import sttp.tapir.server.interceptor.log.{DefaultServerLog, ServerLog}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class NuAkkaHttpServerInterpreterForTapirPurposes(implicit val executionContext: ExecutionContext)
+  extends AkkaHttpServerInterpreter
+    with LazyLogging {
+
+  override val akkaHttpServerOptions: AkkaHttpServerOptions =
+    AkkaHttpServerOptions
+      .customiseInterceptors
+      .exceptionHandler(None)
+      .serverLog(noExceptionLoggingServerLog)
+      .options
+
+  private lazy val noExceptionLoggingServerLog: ServerLog[Future] = {
+    def debugLog(msg: String, exOpt: Option[Throwable]): Future[Unit] = Future.successful {
+      exOpt match {
+        case None => logger.debug(msg)
+        case Some(ex) => logger.debug(s"$msg; exception: {}", ex)
+      }
+    }
+
+    DefaultServerLog[Future](
+      doLogWhenReceived = _ => Future.unit,
+      doLogWhenHandled = debugLog,
+      doLogAllDecodeFailures = (_, _) => Future.unit,
+      doLogExceptions = (_, _) => Future.unit,
+      noLog = Future.unit,
+      logWhenReceived = false,
```

Comment:
Is it ok, that we don't log received, decoding failures and logic exceptions? Why?

Reply:
see the comment above